packages/nimbus/src/components/icon-button/icon-button.mdx

```diff
@@ -14,85 +14,655 @@ figmaLink: >-
   https://www.figma.com/design/AvtPX6g7OGGCRvNlatGOIY/New-Design-System?node-id=58-132&m=dev
 ---
 
-# IconButton
+# Icon Button
 
-displays a button with an icon only as child.
+---
+
+Provides a compact and visually efficient way to represent actions, especially
+when space is limited or the action is universally understood.
+
+## Overview
+
+These specifications define the standard styles and variations for icon buttons
+within our design system, covering different visual treatments for diverse use
+cases.
+
+### Resources
+
+Deep dive on details and access design library.
+
+[TECH RELATED LINK](#) [TECH RELATED LINK](#)
```

Comment:
Going by a feedback I got, we may not need these empty links for now
```suggestion
```

Reply:
Thanks for this, Douglas! Resolved in ee9f6db442f326a9f8846c3d75905c1b2c28cc38